lib/src/flutter_google_places.dart

```diff
@@ -69,7 +69,11 @@ class PlacesAutocompleteWidget extends StatefulWidget {
       this.startText,
       this.debounce,
       this.headers})
-      : super(key: key);
+      : super(key: key) {
+        if (apiKey == null && proxyBaseUrl == null) {
+          throw 'One of `apiKey` and `proxyBaseUrl` fields is required';
```

Comment:
Should throw `ArgumentError` https://api.dart.dev/stable/2.12.2/dart-core/ArgumentError-class.html instead of a String :))